runhouse/servers/http/http_server.py

```diff
@@ -668,7 +680,7 @@ def get_status(request: Request):
     @staticmethod
     def _collect_cluster_stats():
         """Collect cluster metadata and send to Grafana Loki"""
-        if not configs.data_collection_enabled():
+        if not configs.telemetry_collection_enabled():
```

Comment:
shouldn't we be checking this value when doing the periodic cluster checks?

Reply:
yeah, I even think that we should not create the periodic cluster checks thread if `telemetry_collection_enabled()` is `false`. (if the user want to change `telemetry_collection_enabled` to true, he or she will be required to restart the server -> a "new" cluster servlet instance will be created -> periodic cluster checks thread will be created because ` telemetry_collection_enabled = true` )

wdyt?

Reply:
yeah, we are checking it.

btw - it seems that this method is obsolete, looks like we don't use it because it causing latency.